src/test/java/com/wikia/webdriver/common/logging/PageObjectLogging.java

```diff
@@ -246,13 +246,13 @@ public static void stopLogging() {
 
   @Override
   public void beforeNavigateTo(String url, WebDriver driver) {
-    new JavascriptActions(driver).execute("window.stop()");
+//    new JavascriptActions(driver).execute("window.stop()");
     StringBuilder builder = new StringBuilder();
     builder.append(
         "<tr class=\"success\"><td>Navigate to</td><td>" + "<a href='" + url + "'>" + url
         + "</a></td><td> <br/> &nbsp;</td></tr>");
     CommonUtils.appendTextToFile(logPath, builder.toString());
-    logJSError(driver);
+//    logJSError(driver);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3ACommentedOutCodeLine)